release_notes/unreleased.md

```diff
@@ -1 +1,5 @@
-**Unreleased**
+* [PAPP-34630]
```

Comment:
Just the PAPP number will be a bit ambiguous in the public changelog. Could you extend it a bit or add it at the end of the changes listed below if these are related?

Reply:
Updated the release notes to include the specific PAPP number for each change